lib/async_hooks.js

```diff
@@ -238,12 +238,14 @@ class AsyncResource {
     }
     ObjectDefineProperties(bound, {
       'length': {
+        __proto__: null,
```

Comment:
this is fine for consistency, but it's not needed whenever a descriptor is "complete" - ie, four values.

Reply:
I don't think that's correct:

```js
Object.prototype.get = 'whatever'
Object.defineProperty({}, 'key', {
  configurable: true,
        enumerable: false,
        value: fn.length,
        writable: false,
})
```

Reply:
ahh, fair point, because it checks for an "invalid" descriptor. nvm.